### DISCUSSION TITLE
ASLEvent how to string

### DISCUSSION THREAD

**Discussion by Dragonoftheworld** [textadventures/quest#1272] [Creating games with Quest] [answered]
Heyall

So I have this:
```
s = "▸ I don't know."
JS.eval ("setTimeout(function () {ASLEvent('IntroText', " + s + ")}, 800);")
```
IntroText function puts s in 'msg'. I'm doing this because Quest doesn't do sub-1s timers. 
It doesn't work. However this does:
```
JS.eval ("setTimeout(function () {ASLEvent('IntroText', \"▸ I don't know.\")}, 800);")
```
Is there something in ASLEvent that I don't know? 🤔

**Comment by Pertex** [comment, marked as answer]
But you can do this: 
```
s = "\"▸ I don't know.\""
``` 

It's the same what you do with
```
JS.eval ("setTimeout(function () {ASLEvent('IntroText', \"▸ I don't know.\")}, 800);")
```

**Reply by Dragonoftheworld** [reply]
Oops. Thank you

**Reply by KVonGit** [reply]
I just learned why half my code doesn't work when I attempt things like this.
